video_manipulation_functions/video_manipulation.py

```diff
@@ -0,0 +1,101 @@
+import cv2
+import shutil
+import os
+from BionicEye.video_manipulation_functions.given_functions import generate_metadata, is_frame_tagged
+from BionicEye.app import Video, Metadata, Frame
+from BionicEye.singelton_classes.db_manager import DBManage
+from BionicEye.singelton_classes.os_manager import OSManage
+
+DB_MANAGE = DBManage()
+OS_MANAGE = OSManage()
+
+
+def count_frames(video_path):
+    """
+    Counts frames in a video
+    :param video_path: path to a video file
+    :return: the amount of frames in the video
+    """
+    frame_number = 0
+    cap = cv2.VideoCapture(video_path)
+    read_correctly, frame = cap.read()
+
+    while read_correctly:
+        frame_number += 1
+        read_correctly, frame = cap.read()
+
+    return frame_number
+
+
+def save_video(video_path):
+    """
+    Saves given video information to the db and the os if not exists already
+    :param video_path: path to the video file we want to save
+    """
+    video_os_path = os.path.relpath(video_path)
```

Comment:
You're using relpath twice, once here and again inside to `OS_MANAGE.upload_file`

Reply:
because I need to save it also to the db